Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG AWS_CLI_VERSION=2.1.29
 FROM hashicorp/terraform:$TERRAFORM_VERSION as terraform
 
 FROM amazon/aws-cli:$AWS_CLI_VERSION
-ARG KUBECTL_VERSION=1.19.9
+ARG KUBECTL_VERSION=1.21.2
```

Comment:
Given this is in the docker file and any 1.21 version would support the AWS 1.21 version, which happens to be 1.21.2, why would we not try and keep these the same across all cloud providers? Recently we updated Azure to used kubectl 1.21.7 I believe shown [here](https://github.com/sassoftware/viya4-iac-azure/pull/217/files).